changes-set.txt

```diff
@@ -99,6 +99,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+18-Aug-24 difexd    [same]      moved from SN's mathbox to main set.mm
```

Comment:
So this also includes the move of ~difexd, same as #4139 ?

Reply:
Maybe this will lead to merge conflicts.

Reply:
> So this also includes the move of ~difexd, same as #4139 ?

No, #4139 is about ~ifexd, not ~difexd.

Reply:
Oh, of course. Names were so similar, but it's completely different.